content/_releases/22/32.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Product Update - v22.32
+description: Details of the product update v22.32 from 11th of August 2022.
+layout: rel
+version: "22.32"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-08-11
+redirect_from:
+  - /releases/22.32.html
+---
+
+
+## Improvements and Updates
+
+### Continued improvements
+
+Here we should talk about the automated vulnerabilities checks of our code.
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+### Scalability of services
+
+elastic.io platform runs many services under the hood like `webhooks`, `api`, `frontend`, etc.
+Most of these services can and must run in replication meaning they can be scalled to suit
+the needs and the load. Some services, however, do not support scalability due to
+their nature. In this update we removed possibility to define scalability for `bloodyGate`
+and `handmaiden` services from the HELM3 charts configurations.
+
+## OEM Related
+
+Information in this section is intended for our customers who use the
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### Contract level custom metadata
+
+We introduce a new possibility to add a custom metadata to the contracts. A new
+section called `custom_data` can be used to store non-sensitive string data up-to 1MB in
+size. To add, edit and read contract custom metadata you need to have
+`contracts.contract.edit_custom_data` permission (`tenantAdmin` has this permission
+by default) to perform `POST`, `PATCH` and `GET` calls to the `/v2/contract` API endpoint.
+
```

Comment:
This new feature enables Tenant Owners to better differentiate different types of Contracts they supply to their Clients and Users. For instance, a selection of different Contract types can be defined and automatically created, deployed, and maintained using this metadata as basis. Each contract type can have a wide range of specified features, such as GB RAM quota limits, numbers of users or workspaces, and even connector whitelists.